vuepress/docs/next/tutorials/compose/digital-assets-tutorial.md

```diff
@@ -1,136 +1,101 @@
+4. Drag and Drop your files into the window or click `Browse`. The file you choose will appear in the Name field. You can choose multiple files at one time. You can also use the `Add Another Resource` button and `Choose file` individually.
+5. After all files have been chosen, click the `Add` button to upload the files. This takes you back to the main assets page where your uploaded files will be displayed in the table. If you have an image file, see Edit Assets below. 
+6. If a Name field is left blank, you will receive an error message. Simply go to the actions (indicated by three vertical dots) menu next to the empty file and select Delete. This action menu also provides an edit option for each resource.
 
-### Cropping Images
-You can crop images newly uploaded via Digital Assets. You can either crop the image dynamically or by using pre-configured crop ratios.
+## Edit Assets 
+Changes to the Asset name and categories can be made in the Edit mode.  
+1. From `Content` → `Assets` page, in the list of assets, click on the Asset's `Actions` button and select `Edit`. 
+2. Edit the Name field. For Categories, use the `+` button under `Join` to add the asset to a category. 
+3. Once all changes are done, click the `Add` button to finish. 
+  
+To organize or download assets, go to `Administration` → `File browser` and navigate to `/public/cms` to find your documents and images.
 
-To set pre-configured crop ratios:
-- Select `CMS → Content Settings`
-- Under `Settings Image Crop Dimensions,` click *Add crop dimension*.
-- Enter a crop ratio, using numeric values separated by a colon.
-- Click *Save*.
+## Edit Images
+You can flip, rotate and crop newly uploaded images.
 
-To dynamically crop an image:
-- Edit the asset by opening *Actions* menu > Edit.
-  - Buttons representing your pre-defined crop ratios are below the image. Clicking any of those buttons changes the selected crop area based on the chosen ratio.
-  - You can create multiple crops of a single image as part of the upload process. The right side of the screen displays the list of crops created during the current session.
+1. Go to `Content` → `Assets` → `Add` 
+2. After uploading an image, click on the actions button to the right of the image. From the drop-down menu, Select Edit.
+![Add Digital Assets Window](./img/assets-edit-image.png)
+3. Use the arrows to flip, rotate or recenter your image. When done, click the `Crop` check button. Multiple resized images can be created at this time. Click `Free` to remove resizing ratio restrictions.
+4. Click `Done`.
+5. Click Add to save all versions.
 
-![image](./img/cms_digital_assets_crop.png)
+## Embed an Asset on a Page
+The uploaded asset can be embedded on a page by using `<@wp.resourceUrl>your-path/your-asset.jpg`. The path can be found in the App Builder at `Administration` → `File Browser`. (e.g. 
```

Comment:
@nshaw should there be a space between '>' and 'your-path/...'?
<@wp.resourceUrl>your-path/your-asset.jpg

Reply:
another citation in the doc had a space so checking